packages/amplication-data-service-generator/src/server/resource/resolver/resolver.base.template.ts

```diff
@@ -15,6 +15,8 @@ import { isRecordNotFoundError } from "../../prisma.util";
 // @ts-ignore
 import { MetaQueryPayload } from "../../util/MetaQueryPayload";
 // @ts-ignore
+import { Public } from "../../decorators/public.decorator";
+// @ts-ignore
```

Comment:
since we are adding this conditionally from the code, should we remove this?

Reply:
@yuval-hazaz the implementation of adding an import to 'Public' decorator is only in the create-controller for now. I decided to wait until you approve it and then merge it and use it in the create-resolver file. I will add the implementation now and remove this line